src/pages/Ranking/index.js

```diff
@@ -14,73 +15,108 @@ import './styles.css';
 const WORD_HEIGHT = 40;
 
 const Ranking = () => {
-  const { currentUser, currentUserPlayed, dailyResults, expandedUser, setExpandedUser } =
-    useRankingData();
+  const { push } = useHistory();
+
+  const {
+    currentUser,
+    currentUserPlayed,
+    currentStreakRankingData,
+    dailyResults,
+    expandedUser,
+    setExpandedUser,
+    loading,
+  } = useRankingData();
 
   return (
     <div className="ranking">
       <h1 className="section-title">Ranking</h1>
-      <Tabs className="Tabs">
-        <TabList>
-          <Tab>Today</Tab>
-          <Tab>All Time Average</Tab>
-        </TabList>
-        <TabPanel>
-          {dailyResults.map(
-            ({ attemptedWords, attempts, position, status, user: { email, name, photo } }) => {
-              const isCurrentUser = email === currentUser;
-              const lost = status === GAME_STATUS.lost;
-              const isExpanded = email === expandedUser;
-              const isDisabled = !currentUserPlayed;
+      {loading ? (
+        <Loading />
+      ) : (
+        <Tabs className="Tabs">
+          <TabList>
+            <Tab>{`Today (${dailyResults.length})`}</Tab>
+            <Tab>{`Current Streak (${currentStreakRankingData.length})`}</Tab>
+          </TabList>
+          <TabPanel>
+            {dailyResults.map(
+              ({ attemptedWords, attempts, position, status, user: { email, name, photo } }) => {
+                const isCurrentUser = email === currentUser;
+                const lost = status === GAME_STATUS.lost;
+                const isExpanded = email === expandedUser;
```

Comment:
I think it is confusing to have `isExpaned`, `setExpandedUser` and `expandedUser`. Is there any particular reason you need to save the selected user in the state of `useRankingData`? It will limit you to being unable to select multiple users. A better approach would be to move this logic to each row state. WDYT?

Reply:
Yeah, I decided to have only one user expanded at a time but no reason in particular, just a decision. I can totally change it but I will do it in another PR, thanks! 😄